yt_dlp/extractor/mave.py

```diff
@@ -1,24 +1,73 @@
-import re
-
 from .common import InfoExtractor
 from ..utils import (
     clean_html,
     int_or_none,
     parse_iso8601,
-    urljoin,
 )
-from ..utils.traversal import require, traverse_obj
 
 
-class MaveIE(InfoExtractor):
-    _VALID_URL = r'https?://(?P<channel>[\w-]+)\.mave\.digital/(?P<id>ep-\d+)'
+class MaveBaseIE(InfoExtractor):
+    _API_BASE_URL = 'https://api.mave.digital/v1/website'
+    _API_BASE_STORAGE_URL = 'https://store.cloud.mts.ru/mave/'
+
+    def _load_channel_meta(self, channel_id):
+        return self._download_json(
+            f'{self._API_BASE_URL}/{channel_id}/',
+            channel_id)
```

Comment:
As this is purely for metadata I would make it non-fatal, and add a note
```suggestion
        return self._download_json(
            f'{self._API_BASE_URL}/{channel_id}/', channel_id,
            note='Downloading channel metadata', fatal=False)
```